### ISSUE TITLE
[BUG] if I load different manga (chapter) from recommendations panel, SyncDex thinks it still the previous manga

### ISSUE THREAD

**Issue by frykauf** [Glagan/SyncDex#31]
**Describe the bug**
if I load different manga from recommendations panel, SyncDex thinks it still the same manga (tries to update the chapter as a chapter from the previous manga you were reading).

**To Reproduce**
1. reading manga chapter on Mangadex with recommendations feature switched on
2. finishing all available chapters
3. switching to another manga
4. SyncDex tries to update chapter to the previous manga you're reading, not recognizing you're reading different manga

**Expected behavior (optional)**
Properly recognize that you loaded a chapter from different manga (and update read chapters for that manga).

**Screenshots**

**Browser**
Firefox or Chrome (and version)

**Options and logs**
```
Click the **Debug** button on the navigation menu on the left in the Options and copy paste it's content here.
```

**Additional context**

**Comment by Glagan** [comment]
Sorry I'm not using this so I didn't know about this, I'll look at it.

**Comment by Glagan** [comment]
Should be fixed with version ``0.2.1``.
